### PR TITLE
Implementation Order: Add CustomerData to order #69

### PR DESCRIPTION
**Pull request by junetaw**
Order: Add CustomerData to order #69

### CODE REVIEW

File: 2.6.0/spec/orders.md [hunk 1]
```diff
@@ -692,4 +704,78 @@ Can be used in the response to describe why the status is DONE_FAILED
...
 * Required (but may be empty)
- 
\ No newline at end of file
+
+### customer
+
+Can be sent to COs that require customer information.
+
+* Data format: JSON array of JSON objects
+* Mandatory if CO requires customer data, otherwise optional.
```
[the remark this answers is not in the document]
Comment (junetaw) [reply 2.6.0/spec/orders.md:713]: Yes, I will change it to prohibited and add in Note that CO should throw an exception if SP sends customer data that the CO does not want. Note2 is therefore not needed and will be removed.

File: 2.6.0/spec/orders.md [hunk 2]
```diff
@@ -692,4 +704,78 @@ Can be used in the response to describe why the status is DONE_FAILED
...
+
+### customer/socialSecurityNumber
+
+Social security number of the customer
+
+ * Data format: [identitynumber](../common/dataformats.md#identitynumber)
+
+### customer/email
+
+Last name of the customer
```
[the remark this answers is not in the document]
Comment (junetaw) [reply 2.6.0/spec/orders.md]: Fixed

File: 2.6.0/spec/orders.md [hunk 3]
```diff
@@ -692,4 +704,78 @@ Can be used in the response to describe why the status is DONE_FAILED
...
 * Data format: [text](../common/dataformats.md#text)
 * Required (but may be empty)
- 
\ No newline at end of file
+
+### customer
+
+Can be sent to COs that require customer information.
+
+* Data format: JSON array of JSON objects
```
[the remark this answers is not in the document]
Comment (junetaw) [reply 2.6.0/spec/orders.md:714]: I have added organizationNumber and organizationName to the specification. At least two specifications in Order violates the customerDetails specifications: organizationNumber and zipCode.

**Review by teemufrosen — APPROVED** [review]

**Comment by teemufrosen** [comment]
Working group meeting: Needs further discussion, current state of PR poses a GDPR risk.

**Pull request closed without merging**